freeipa.spec.in

```diff
@@ -100,6 +100,16 @@
 %global httpd_version 2.4.41-6.1
 %endif
 
+# BIND in Fedora 31 and later uses 'pkcs11' OpenSSL engine instead of native
+# PKCS11
+%if 0%{?fedora} >= 31
+    %global with_bind_pkcs11 0
+    %global openssl_pkcs11_version 0.4.10-3
```

Comment:
Please require ``0.4.10-6`` with fix for https://bugzilla.redhat.com/show_bug.cgi?id=1827535